.gitignore

```diff
@@ -67,9 +67,9 @@ public/critical.css
 packages/volto/data
 
 # Sphinx and MyST
-docs/_build/
```

Comment:
You committed this directory, but should not have done so.

Reply:
Done.